ansible/donkeycar.yml

```diff
@@ -8,8 +8,8 @@
   roles:
     - role: raspi-config #rasperry pi configuration role
       become: yes # This role need root privileges
```

Comment:
you only need one `# This role need root privileges` comment, the rest of them decrease readability